proto/viam/app/package/v1/package.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+
+package viam.app.package.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/app/package/v1";
+
+service PackageService {
+  // CreatePackage uploads a package to the cloud
+  rpc CreatePackage(stream CreatePackageRequest) returns (CreatePackageResponse) {
+    option (google.api.http) = {
+      post: "/viam/app/package/v1/create"
+    };
+  }
+
+  // DeletePackage removes the given package versions
+  rpc DeletePackage(DeletePackageRequest) returns (DeletePackageResponse) {
+    option (google.api.http) = {
+      delete: "/viam/app/package/v1/delete"
+    };
+  }
+
+  // // GetPackage returns the URL and metadata for a requested package version
+  rpc GetPackage(GetPackageRequest) returns (GetPackageResponse) {
+    option (google.api.http) = {
+      get: "/viam/app/package/v1/package"
+    };
+  }
+
+  // ListPackages get the URLs and metadata for requested packages. Include package name, version, and/or type to filter beyond the required organization_id.
+  rpc ListPackages(ListPackagesRequest) returns (ListPackagesResponse) {
+    option (google.api.http) = {
+      get: "/viam/app/package/v1/packages"
```

Comment:
nit: maybe `/package/v1/list`? open to whatever for these, just trying to stick to a convention where the last bit is a verb like 'create' or 'delete'

Reply:
done :) thanks!